README.md

```diff
@@ -42,83 +24,58 @@ The EzyFox ecosystem supports a wide range of essential components for enterpris
 
 With the EzyFox ecosystem, we can avoid the headache of choosing which technologies to use to manage and scale up an application, allowing us to focus solely on implementing business logic.
 
-# Getting Started
-
-[https://youngmonkeys.org/get-started/](https://youngmonkeys.org/get-started/)
-
-# Documentation
+## ✨ Features
 
-[https://youngmonkeys.org/ezyfox-sever/](https://youngmonkeys.org/project/ezyfox-sever/)
-
-# Latest Version
+* **Core Container & Dependency Injection**: Includes bean manipulation, auto-binding, auto-implementation, and more.
+* **Multiple Communication Protocols**: Supports TCP, UDP, WebSocket, and HTTP.
+* **Traffic Encryption**: Allows for traffic between clients and servers to be encrypted using SSL.
+* **Multiple Client SDKs**: Includes Android, iOS, Unity, React, C++, Flutter, and more.
 
-You can [download it here](https://resources.tvd12.com/)
 
-# Code Example
+## 🚀 Getting Started
 
-**1. Create an app entry**
+[Let's see how we can get started in just 5 minutes!](https://youngmonkeys.org/get-started/)
 
-```java
-public static class HelloAppEntry extends EzySimpleAppEntry {
+## 📑 Documentation
 
-    @Override
-    protected String[] getScanablePackages() {
-        return new String[] {
-                "com.tvd12.ezyfoxserver.embedded.test" // replace by your package
-        };
-    }
-}
-```
+- [Official website document](https://youngmonkeys.org/projects/ezyfox-server)
+- [Issue](https://github.com/youngmonkeys/ezyfox-server/issues)
```

Comment:
```suggestion
- [Issues](https://github.com/youngmonkeys/ezyfox-server/issues)
```

Reply:
@TrungKenbi A couple of final suggestions, thanks

Reply:
Finally, I updated it all. Many thanks for your suggestion @vu-luong